tests/fixtures/certs/es256.pub

```diff
@@ -29,18 +29,3 @@ BBQOfI1yZr+iyeUAlO+1boDitg4vazAfBgNVHSMEGDAWgBRembiG4Xgb2VcVWnUA
 UrYpDsuojDAKBggqhkjOPQQDAgNJADBGAiEAtdZ3+05CzFo90fWeZ4woeJcNQC4B
 84Ill3YeZVvR8ZECIQDVRdha1xEDKuNTAManY0zthSosfXcvLnZui1A/y/DYeg==
 -----END CERTIFICATE-----
```

Comment:
@dyro Why/How did that change? Is it because I ran a build locally?

Reply:
I don't know - that shouldn't change automatically. Maybe revert this change?